### DISCUSSION TITLE
为什么官方文档写得那么晦涩，是写给谁看的呀

### DISCUSSION THREAD

**Discussion by wangwufan** [antvis/G2#5057] [General]
![image](https://github.com/antvis/G2/assets/12657738/aea04f7b-5546-4f00-b1b7-1534a343ee5b)

**Comment by hustcc** [comment]
有什么建议吗？

基于图形语法，相比来说，确实文档比较难组织，而且上手门槛也会高一些，需要大概了解图形语法的理论知识，如果只是基本图表的使用，建议可以使用 [G2Plot](https://g2plot.antv.antgroup.com/)，会相对来说容易很多。

**Reply by x011223** [reply]
g2 plot 以及 bizcharts 都是基于 v4，并且长时间没有发版，实在有些怵的慌...

**Reply by hustcc** [reply]
当前正在升级 Ant Design Charts 了，有兴趣也可以参入进来，见 issue https://github.com/antvis/G2/issues/5408 。
